### PR TITLE
Revamped the UG and fixed a minor CSS issue

### LINKED ISSUE #121: Appointment and Person cards highlight issues
The Person card should not be highlighted anymore after resetting. The Appointcard does not have the highlight feature yet. I will fix these.

### LINKED ISSUE #135: [PE-D][Tester B] Lack of test input example
Example inputs for test cases not provided for most of the commands which for new users can be difficult to comprehend the command format structure and input data to test in quick manner.

Would be nice to add some example inputs in which readers/users can use to test and get a quick start to the application.

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: Zack-Tay/ped#4

### LINKED ISSUE #187: [PE-D][Tester A] DATE parameter for adding patients accepts undocumented input formats
**Steps to reproduce:**
1) Add a patient using the following commands:
- `add n/Date test p/12345678 d/tody 10pm-11pm`
- `add n/Date test2 p/12345678 d/tday 10pm-11pm`

**Expected:**
Invalid command format is thrown due to the format for DATE parameter not adhering to what was shown in the user guide.

**Actual:**
Patient entries are created with the date set as the current day.

**Improvement suggestion:**
Might want to update the user guide to reflect these acceptable variations to the DATE parameter format or invalidate them as accepted inputs for DATE.

There is another bug captured in the screenshot below where the time of the appointment is not reflecting the time that was used as input. More information about this bug can be found in the bug report titled: 'Incorrect appointment time is displayed when 'd/tody 10pm-11pm/10am-11am' is used as DATE_TIME input'.

**Screenshot:**

-------------
Labels: `type.FunctionalityBug` `severity.VeryLow`
original: JonChong98/ped#13

### PR DESCRIPTION
**Pull request by legionlegion**
Fixes #187 - now the formats for today are documented to be /tdy and /today only
Fixes #135 - now examples are all documented
Fixes #121 - this is now a known issue and will not be fixed for now

Some potential issues for our UG, someone please follow up:
1. What's the ID parameter for findappt?
image
2. We place screenshots inconsistently for the commands in our UG. Should we include 1 screenshot for every command?

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -50,64 +48,66 @@ The clinical workflow is separated into three steps with RapidTracer:
   1. Input parameters
 </h3>
 
-The table below lists all input parameters and their max length.
+The table below lists all input parameters and their possble values.
```
Comment (chengjunyuan) [inline docs/UserGuide.md:51]: Possible

File: docs/UserGuide.md [hunk 2]
```diff
@@ -142,48 +140,47 @@ RapidTracer offers search functions for both patient and appointment data. Beyon
...
 
-- The KEYWORD searched is case-insensitive.
+- The `KEYWORD` search is case-insensitive.
 - Order of keywords does not matter.
-- Possible keyword types:
-  - NAME
-  - ID
+- Searches for the following appointment information:
+  - `NAME`
+  - `ID` (what's this?)
```
Comment (chengjunyuan) [inline docs/UserGuide.md:168]: I think can delete `ID`, the patient UUID was originally meant to be human readable but this was changed

**Review by chengjunyuan — APPROVED** [review]
LGTM. Can look at the minor changes

**Comment by legionlegion** [comment]
@chengjunyuan can review again? I updated the branch with you changes, should be good to go without having to read through again

**Review by chengjunyuan — APPROVED** [review]
LGTM

**Pull request merged**